Odata-docs/webapi-8/fundamentals/overview.md

```diff
@@ -0,0 +1,582 @@
+---
+title: "ASP.NET Core OData 8 Fundamentals Overview"
+description: "This document introduces the fundamental concepts of ASP.NET Core OData library."
+
+author: habbes
+ms.author: clhabins
+
+ms.date: 1/5/2023
+---
+# Fundamentals Overview
+**Applies To**:[!INCLUDE[appliesto-webapi](../../includes/appliesto-webapi-v8.md)]
+
+This article provides an overview of the fundamental concepts for building ASP.NET Core OData services.
+
+ASP.NET Core OData is a .NET library built on top of ASP.NET Core to help you build OData REST APIs. The ASP.NET Core OData library consists of a set of services and middleware that hook into ASP.NET Core's request pipeline to provide features like routing, query handling and serialization based on OData specifications and conventions. All these concepts will be covered in depth in separate articles, this article just provides an overview.
+
+To build an ASP.NET Core OData app, you start with an ASP.NET Core application, then install the [`Microsoft.AspNetCore.OData`](https://www.nuget.org/packages/Microsoft.AspNetCore.OData) NuGet package as a dependency. ASP.NET Core OData 8 supports .NET Core 3.1 and .NET Core 6.0 and above.
+
+> [!NOTE]
```

Comment:
When I saw this note, I thought it's used to emphasize a statement from the body of the doc. Then I went further down and came across a note that doesn't reference a statement from the body of the doc. What is the practice?

Reply:
This is the use case from the NOTE alert according to the [guidelines](https://learn.microsoft.com/en-us/contribute/markdown-reference#alerts-note-tip-important-caution-warning): Information the user should notice even if skimming.

My aim was to emphasize something that I thought was important, especially important "gotchas" that could affect the developer experience if not taken into account. But the guidelines discourages frequent use of alerts within the same docs:

> Avoid notes, tips, and important boxes. Readers tend to skip over them. It's better to put that info directly into the article text.

> If you need to use alerts, limit them to one or two per article. Multiple notes should never be next to each other in an article.

So let me remove the note since it's already referenced in the docs.